src/renderer/api/json-api.ts

```diff
@@ -64,7 +64,7 @@ export class JsonApi<D = JsonApiData, P extends JsonApiParams = JsonApiParams> {
   }
 
   patch<T = D>(path: string, params?: P, reqInit: RequestInit = {}) {
-    return this.request<T>(path, params, { ...reqInit, method: "patch" });
+    return this.request<T>(path, params, { ...reqInit, method: "PATCH" });
```

Comment:
https://stackoverflow.com/questions/34666680/fetch-patch-request-is-not-allowed
Method PATCH is case sensitive